pkg/eks/tasks.go

```diff
@@ -117,7 +117,7 @@ func (c *ClusterProvider) CreateExtraClusterConfigTasks(cfg *api.ClusterConfig,
 	}
 	if !cfg.HasClusterCloudWatchLogging() {
 		logger.Info("CloudWatch logging will not be enabled for cluster %q in %q", cfg.Metadata.Name, cfg.Metadata.Region)
-		logger.Info("you can enable it with 'eksctl utils update-cluster-logging --region=%s --cluster=%s'", cfg.Metadata.Region, cfg.Metadata.Name)
+		logger.Info("you can enable it with 'eksctl utils update-cluster-logging --enable-types=all --region=%s --cluster=%s'", cfg.Metadata.Region, cfg.Metadata.Name)
```

Comment:
I think the idea might be to force the user to see the error message and choose for themselves what logging should be enabled. What do you think?

Reply:
OK, then I would personally prefer to add "--enable-types={SPECIFY-LOG-TYPES}" or something like that....
Cause the error message after executing the command does not immediately imply the correct usage (without consulting the help).
But indeed it is also one of the many ways that lead to Rome ... ;-)